packages/convex-helpers/README.md

```diff
@@ -309,3 +327,37 @@ const balanceAndEmail = pick(Account.withoutSystemFields, ["balance", "email"]);
 // A validator for all the fields except balance.
 const accountWithoutBalance = omit(Account.withSystemFields, ["balance"]);
 ```
+
+## Filter
+
+See the [guide on Stack](https://stack.convex.dev/complex-filters-in-convex)
+for an analysis of complex filters on Convex.
+
+The `filter` helper composes with `ctx.db.query` to apply arbitrary TypeScript
+or JavaScript filters to a database query.
+
+Examples:
+
+```javascript
```

Comment:
can you include the import statement so it's easy to copy? It's often not clear where to import things from and auto-complete for imports hasn't been working well for me for helpers for some reason